detox/src/client/actions/actions.js

```diff
@@ -306,7 +306,7 @@ class GetViewHierarchyXml extends Action {
   }
 
   async handle(response) {
-    return response.params.viewHierarchy;
+    return { 'result': response.params.viewHierarchy };
```

Comment:
So, what do we return to the user? Isn't it just the view hierarchy or { result:..} object?